<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>770</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1230</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 12, 44);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Commands">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>700</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(48,44,68,150);
color: white;
border: 1px solid black; 
border-radius: 20px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendButton">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>700</y>
      <width>136</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black; 
border-radius: 10px;
background-color:rgb(68,68,108);
background-color:rgb(52,68,204);
</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="Footer">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>740</y>
      <width>1261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:rgb(24,20,52);
font: 13pt &quot;Georgia&quot;;
color:rgb(68,68,108);
border: 1px solid black; 
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>        Ground-Station                                                                                                                                                                                                                                                                                                   Institute for Earth Observations</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="ByteData">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>511</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(24,20,52);
border: 1px solid black; 
border-radius: 15px;
lineWrapColumnOrWidth=320;
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ProcessedData">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>511</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(24,20,52);
border: 1px solid black; 
border-radius: 15px;
lineWrapColumnOrWidth=300;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Header">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1251</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(24,20,52);
font: 13pt &quot;Georgia&quot;;
border: 1px solid black; 
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="SpeedStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>136</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(113,51,253,255);
border: 1px solid black; 
lineWrapColumnOrWidth=91;
border-radius: 15px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:15pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:696;&quot;&gt;Speed&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="AltitudeStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>136</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(113,51,253,255);
border: 1px solid black; 
lineWrapColumnOrWidth=91;
border-radius: 15px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:15pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:696;&quot;&gt;Altitude&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DistanceStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>136</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(113,51,253,255);
border: 1px solid black; 
lineWrapColumnOrWidth=91;
border-radius: 15px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:15pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:696;&quot;&gt;Temperature&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>150</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black; 
border-radius: 10px;
background-color:rgb(68,68,108);
background-color:rgb(36,116,116)</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(24,20,52);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IEO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="DistanceDisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(113,51,253,255);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="AltitudeDisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(113,51,253,255);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedDisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(113,51,253,255);
border-radius: 15px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Terminal">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>449</width>
      <height>581</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>309</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(9,3,14,255);
border: 1px solid black; 
border-radius: 10px;
lineWrapColumnOrWidth=311;
</string>
    </property>
   </widget>
   <zorder>SpeedStatus</zorder>
   <zorder>AltitudeStatus</zorder>
   <zorder>Commands</zorder>
   <zorder>SendButton</zorder>
   <zorder>Footer</zorder>
   <zorder>ByteData</zorder>
   <zorder>ProcessedData</zorder>
   <zorder>Header</zorder>
   <zorder>DistanceStatus</zorder>
   <zorder>ConnectButton</zorder>
   <zorder>Logo</zorder>
   <zorder>DistanceDisplay</zorder>
   <zorder>AltitudeDisplay</zorder>
   <zorder>SpeedDisplay</zorder>
   <zorder>Terminal</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
